src/layouts/bartoszWrapper.js

```diff
@@ -0,0 +1,17 @@
+import ChooseGameSection from './ChooseGameSection';
+import GameSection from './GameSection';
+import OptionsSection from './OptionsSection';
+import MenuArea from './MenuArea';
+
+const Wrapper = () => {
+  const wrapper = document.createElement('div');
+  wrapper.classList.add('wrapper');
+  wrapper.id = 'wrapper';
+
+  wrapper.appendChild(OptionsSection());
+  wrapper.appendChild(MenuArea());
+  wrapper.appendChild(ChooseGameSection());
+  wrapper.appendChild(GameSection());
```

Comment:
Gdzie jest  część na logo?
Może  skoro góra jest stała to ustawić to w ten sposób:
```
<div>
  <div> // upperSection
     <div> // logo sectioon
        LOGO
     </div>
     <div>// nav sectioon
        MENU
      </div>
    </div>
    <div> // game/main section
          
     </div>
</div>
```

Reply:
ChooseGameSection = kategorie,
OptionsSection = logo, 
GameSection = to co na jest na dole.
Myślę, że jak zmienić nazwy na takie jak zaproponowałem wyżej i wywalić te MenuArea to będzie dużo bardziej czytelne.